dsl/src/main/scala/com/crobox/clickhouse/dsl/column/ComparisonFunctions.scala

```diff
@@ -2,26 +2,20 @@ package com.crobox.clickhouse.dsl.column
 
 import com.crobox.clickhouse.dsl.{EmptyColumn, ExpressionColumn}
 
-trait ComparisonFunctions { self: Magnets =>
+trait ComparisonFunctions {
+  self: Magnets =>
 
   case class ComparisonColumn(left: Magnet[_], operator: String, right: Magnet[_]) extends ExpressionColumn[Boolean](EmptyColumn)
 
-  trait ComparableWith[M <: Magnet[_]] { self: Magnet[_] =>
-    def <(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "<", other)
-    def >(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, ">", other)
-    def <>(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "!=", other)
-    def isEq(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "=", other)
-    def notEq(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "!=", other)
-    def ===(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "=", other)
-    def !==(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "!=", other)
-    def <=(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, "<=", other)
-    def >=(other: M): ExpressionColumn[Boolean] = ComparisonColumn(self, ">=", other)
-  }
-
-  def _equals(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1 , "=", col2)
-  def notEquals(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1 , "!=", col2)
-  def less(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1 , "<", col2)
-  def greater(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1 , ">", col2)
-  def lessOrEquals(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1 , "<=", col2)
-  def greaterOrEquals(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1 , ">=", col2)
+  def _equals(col1: ConstOrColMagnet[_], col2: ConstOrColMagnet[_]): ExpressionColumn[Boolean] = ComparisonColumn(col1, "=", col2)
```

Comment:
I really don't like the _ (which was already there btw). Can't we get rid of it? Or make this one deprecated and add the 'equals()' method. If compiler starts complaining, perhaps rename it to: 'isEquals' or something similar?